### PR TITLE
Pull data from upstream repositories

### PR DESCRIPTION
**Pull request by jayvdb**
Moves gci git code into community app module gci.
Adds a new view info.txt to show backend build info.

Closes https://github.com/coala/community/issues/104

### CODE REVIEW

**Comment by jayvdb** [comment]
Note you can see this as a green build on my fork: https://travis-ci.org/jayvdb/community/builds/354716036

File: data/management/commands/import_contributors_data.py [hunk 1]
```diff
@@ -37,13 +39,19 @@ def handle(self, *args, **options):
...
-        response.raise_for_status()
+        try:
+            response = requests.get(
+                url=IMPORT_URL,
+                headers=headers,
+            )
+            response.raise_for_status()
+        except Exception as e:
+            logger.error(e)
+            return
```
Comment (sks444) [inline data/management/commands/import_contributors_data.py]: `return`? seems very unusual.

Comment (jayvdb) [reply data/management/commands/import_contributors_data.py]: What should a django `Command.handle` do when there is an error ?

**Comment by RaiVaibhav** [comment]
https://deploy-preview-107--coala-community.netlify.com/ 

Netlify

**Comment by hemangsk** [comment]
ack ca2cf71 dcdbcef

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**